### PR TITLE
[DYN-7436] Dynamo crashes upon exporting results if the destination file is open

### PR DESCRIPTION
**Pull request by ivaylo-matov**
Aims to address https://jira.autodesk.com/browse/DYN-7436

The code now check if the file is in use before trying to override it. Please let me know if this is acceptable and can implement it on `Node Graph Manager`.

DYN-7436-fix

@QilongTang 
@reddyashish 

@Amoursol 
@dnenov

### CODE REVIEW

**Review by reddyashish — APPROVED** [review]

**Comment by reddyashish** [comment]
@ivaylo-matov Changes look good, we can do the same for Graph node manager.

File: TuneUp/TuneUpWindowViewModel.cs [hunk 1]
```diff
@@ -1187,54 +1187,95 @@ public void ExportToCsv()
 
             if (saveFileDialog.ShowDialog() == true)
             {
-                using (var writer = new StreamWriter(saveFileDialog.FileName))
+                // Check if the .csv file locked or in use
+                if (IsFileLocked(new FileInfo(saveFileDialog.FileName)))
                 {
-                    writer.WriteLine("Execution Order,Name,Execution Time (ms)");
+                    MessageBox.Show("The file is currently in use by another application. Please close the file before trying to overwrite it.",
```
Comment (reddyashish) [inline TuneUp/TuneUpWindowViewModel.cs:1193]: @ivaylo-matov can you move these messages to resources so that they will be localized. You can do it in a different PR.

Comment (ivaylo-matov) [reply TuneUp/TuneUpWindowViewModel.cs:1193]: Now addressed in this PR : https://github.com/DynamoDS/TuneUp/pull/70
